### DISCUSSION TITLE
Problem upgrading to LME 2.0

### DISCUSSION THREAD

**Discussion by AJohnsonFNSB** [cisagov/LME#537] [General]
I am having an issue while trying to export the indices as a part of the Upgrading from 1x to 2x process. It just sits after the node:7 warning and does not progress. I have tried rebooting the 2 machines and the issue persists. Does anyone have a suggestion on how to get past this?

XXXXXXXX@XXXXXX:~/LME/scripts/upgrade$ sudo ./export_1x.sh
[sudo] password for XXXXXXX:
LME Data Export Script for Elasticsearch 8.x
============================================
Using host IP: X.X.X.X
Enter Elasticsearch username: XXXXXX
Enter Elasticsearch password: XXXXXXX
Successfully connected to Elasticsearch version 8.11.1
Choose backup directory:
1. Specify a directory
2. Automatically find directory with most space
Enter your choice (1 or 2): 1
Enter the backup directory path: /home/XXXXXXXX/LME/scripts/upgrade/exported/
Exporting winlogbeat-* indices data...
(node:7) Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.
(Use `node --trace-warnings ...` to show where the warning was created)

**Comment by cbaxley** [comment]
This can take a super long time. I would open another terminal and make
sure the files are growing in size.
ls -l  /home/XXXXXXXX/LME/scripts/upgrade/exported/

It is just a slow process if you have a lot of logs. It may be the same
with import.

Thanks,
Clint Baxley
